### PR TITLE
Package refresh

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Hello :)

Here is some upgrade for the package!

Was considering, replacing the Makefile with standard npm scripts.
But I prefered to check first if you're okay with it ? :) 
(cc @DonutEspresso @hekike )
If so, tell me. I'll add it to the PR

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
Hello @DonutEspresso @hekike :wave:
Just some message to try to get your attention on this PR :wink:

File: lib/baseClasses/HttpError.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ function HttpError() {
      * @property
      * @type {Function}
      */
-    if (opts.hasOwnProperty('toJSON')) {
+    if (Object.prototype.hasOwnProperty.call(opts, 'toJSON')) {
```
Comment (retrohacker) [inline lib/baseClasses/HttpError.js:93]: Why the conversion to `.call`?

Comment (AdrieanKhisbe) [reply lib/baseClasses/HttpError.js:93]: It is the autofix of following eslint rule [`no-prototype-builtins`](https://eslint.org/docs/rules/no-prototype-builtins), that got added into `eslint:recommended` config

**Comment by AdrieanKhisbe** [comment]
@retrohacker Thanks for the review.
Maybe you'll be interested if I rebase and refresh the branch.
If so, tell me, and I'll do it :)

**Comment by retrohacker** [comment]
@AdrieanKhisbe no need, we use conventional commits so have to squash and merge anyways :smile: 

Thank you for contributing :heart:

**Comment by AdrieanKhisbe** [comment]
@retrohacker Thanks! =)

About the _refreshception_ proposal, that was more to ensure to upgrade dependencies were latest if some were updated since the moment I made the Pull Request :)

Little question, do when you plan to publish ? :)

**Comment by AdrieanKhisbe** [comment]
Hello @retrohacker 
This is a gentle reminder to have a patch/minor bump as you prefer 🙂 

Thanks a lot in advance! :)

**Comment by AdrieanKhisbe** [comment]
👋  @retrohacker @hekike @DonutEspresso.

Some last _"draw attention attempt"_ until summer, so we can have latest `@netflix/nerror` with `restify-errors` :)

Thanks in advance for your time, and really sorry for these messages.

**Pull request merged**